api/users/serializers.py

```diff
@@ -504,7 +502,7 @@ def get_absolute_url(self, obj):
         return obj.absolute_api_v2_url
 
     class Meta:
-        type_ = 'user-education'
+        type_ = 'education'
```

Comment:
Why did you change this value?

Reply:
Upon reconsideration, it makes more sense to have this be `'user-education'` due to the 'UserEducationRelationshipSerializer'